src/session.rs

```diff
@@ -27,8 +28,13 @@ const NACK_MIN_INTERVAL: Duration = Duration::from_millis(100);
 // Delay between reports of TWCC. This is deliberately very low.
 const TWCC_INTERVAL: Duration = Duration::from_millis(100);
 
-const PACING_FACTOR: f64 = 1.1;
-const PADDING_FACTOR: f64 = 1.0;
+// The target bitrate must be slightly above an estimated bitrate
+// so that the pacer tries to "push upwards" when below the
+// desired bitrate.
+const PADDING_OVER: Bitrate = Bitrate::kbps(5);
+
+// Avoid changing the BWE unless the new value is outside some tolerance.
+const ESTIMATE_TOLERANCE: f64 = 0.2;
```

Comment:
sounds very high, purely based past testing experience

Reply:
How do you mean?

Reply:
one can only use 20% of the estimate to avoid overuse

Reply:
The idea is that we don't emit a new BWE event unless we get a change that is 0.2 up or down from the last? Is that what you mean?

Reply:
Yes. So with an estimate of 1 megabit, I can be sending at 900kbps and be overusing, because the network has degraded and I don't know about it.